### ISSUE TITLE
Unable to include Remotec ZRC-90EU and maybe remotes in general?

### ISSUE THREAD

**Issue by SebastianWeb** [athombv/homey#1492]
**What is your Homey version (Settings → System)?**

1.2.2

**What did you try to achieve?**

Include Remotec ZRC-90EU, http://products.z-wavealliance.org/products/1584

**What was the result?**

Inclusion getting stuck at "Retrieving manufacturer data" and then "No z-wave device was found" shows up a few seconds later.

**What did you expect as result?**

A successfull inclusion. The device to be identified as a general z-wave device until I had written a driver for it.

**Any other remarks**

Tried 10 times, same result always. Tried a reboot. Tried to reset Remotec.

**Please attach a Homey System log (Settings -> System -> Send Log)**

4ECB67C00A

**Comment by TedTolboom** [comment]
Should be possible to add it.. 
will check it today when I have a scene master myself and build a driver for it

**Comment by jeroenvollenbrock** [comment]
Could you send us an excerpt of the Z-Wave log as well?

**Comment by TedTolboom** [comment]
@jeroenvollenbrock looks like the node interview is timing out / getting stuck on a interview command or the device is going back into sleep mode... each time communication stops at COMMAND_CLASS_BATTERY, which corresponds to 20 seconds between ADD_NODE_STATUS_NODE_FOUND for multiple occasions. 

Interview is timing out, but node is added as Unknown Z-wave device

Homey Log: BD66B85541
http://products.z-wavealliance.org/products/1584

Barebone app and driver are located at: https://github.com/TedTolboom/hk.com.remotec

See below Z-wave log
```
[2017-04-24T15:18:33.748Z] Command[449] start: addNode
[2017-04-24T15:18:33.819Z] Command[449] status: addNode, status: ADD_NODE_STATUS_LEARN_READY
[2017-04-24T15:18:36.219Z] Command[449] status: addNode, status: ADD_NODE_STATUS_NODE_FOUND
[2017-04-24T15:18:36.709Z] Command[449] status: addNode, status: ADD_NODE_STATUS_ADDING_CONTROLLER
[2017-04-24T15:18:38.895Z] Command[449] status: addNode, status: ADD_NODE_STATUS_PROTOCOL_DONE
[2017-04-24T15:18:38.997Z] Command[449] status: addNode, status: ADD_NODE_STATUS_DONE
[2017-04-24T15:18:39.684Z] Command[449] status: addNode, status: INTERVIEW_CC
[2017-04-24T15:18:39.698Z] Node[53]: sendData to COMMAND_CLASS_VERSION, params 0x135e
[2017-04-24T15:18:39.698Z] Command[450] start: sendData
[2017-04-24T15:18:39.704Z] ProcessSendData[427]: To node: 53 with data: 0x86135e and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:18:39.780Z] Command[450] end: sendData
[2017-04-24T15:18:39.799Z] Node[53]: Received application command for COMMAND_CLASS_VERSION, data: 0x145e02
[2017-04-24T15:18:39.803Z] Command[449] status: addNode, status: INTERVIEW_CC_1_OF_13
[2017-04-24T15:18:39.805Z] Node[53]: sendData to COMMAND_CLASS_VERSION, params 0x1385
[2017-04-24T15:18:39.805Z] Command[451] start: sendData
[2017-04-24T15:18:39.810Z] ProcessSendData[428]: To node: 53 with data: 0x861385 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:18:39.847Z] Command[451] end: sendData
[2017-04-24T15:18:39.868Z] Node[53]: Received application command for COMMAND_CLASS_VERSION, data: 0x148502
[2017-04-24T15:18:39.870Z] Command[449] status: addNode, status: INTERVIEW_CC_2_OF_13
[2017-04-24T15:18:39.872Z] Node[53]: sendData to COMMAND_CLASS_VERSION, params 0x1372
[2017-04-24T15:18:39.872Z] Command[452] start: sendData
[2017-04-24T15:18:39.878Z] ProcessSendData[429]: To node: 53 with data: 0x861372 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:18:39.908Z] Command[452] end: sendData
[2017-04-24T15:18:39.932Z] Node[53]: Received application command for COMMAND_CLASS_VERSION, data: 0x147202
[2017-04-24T15:18:39.935Z] Command[449] status: addNode, status: INTERVIEW_CC_3_OF_13
[2017-04-24T15:18:39.936Z] Node[53]: sendData to COMMAND_CLASS_VERSION, params 0x1321
[2017-04-24T15:18:39.936Z] Command[453] start: sendData
[2017-04-24T15:18:39.943Z] ProcessSendData[430]: To node: 53 with data: 0x861321 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:18:39.997Z] Command[453] end: sendData
[2017-04-24T15:18:40.022Z] Node[53]: Received application command for COMMAND_CLASS_VERSION, data: 0x142101
[2017-04-24T15:18:40.025Z] Command[449] status: addNode, status: INTERVIEW_CC_4_OF_13
[2017-04-24T15:18:40.025Z] Node[53]: sendData to COMMAND_CLASS_VERSION, params 0x1384
[2017-04-24T15:18:40.026Z] Command[454] start: sendData
[2017-04-24T15:18:40.031Z] ProcessSendData[431]: To node: 53 with data: 0x861384 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:18:40.077Z] Command[454] end: sendData
[2017-04-24T15:18:40.101Z] Node[53]: Received application command for COMMAND_CLASS_VERSION, data: 0x148402
[2017-04-24T15:18:40.104Z] Command[449] status: addNode, status: INTERVIEW_CC_5_OF_13
[2017-04-24T15:18:40.105Z] Node[53]: sendData to COMMAND_CLASS_VERSION, params 0x1386
[2017-04-24T15:18:40.106Z] Command[455] start: sendData
[2017-04-24T15:18:40.110Z] ProcessSendData[432]: To node: 53 with data: 0x861386 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:18:40.144Z] Command[455] end: sendData
[2017-04-24T15:18:40.163Z] Node[53]: Received application command for COMMAND_CLASS_VERSION, data: 0x148602
[2017-04-24T15:18:40.167Z] Command[449] status: addNode, status: INTERVIEW_CC_6_OF_13
[2017-04-24T15:18:40.168Z] Node[53]: sendData to COMMAND_CLASS_VERSION, params 0x1380
[2017-04-24T15:18:40.168Z] Command[456] start: sendData
[2017-04-24T15:18:40.173Z] ProcessSendData[433]: To node: 53 with data: 0x861380 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:18:40.208Z] Command[456] end: sendData
[2017-04-24T15:18:40.230Z] Node[53]: Received application command for COMMAND_CLASS_VERSION, data: 0x148001
[2017-04-24T15:18:40.233Z] Command[449] status: addNode, status: INTERVIEW_CC_7_OF_13
[2017-04-24T15:18:40.233Z] Node[53]: sendData to COMMAND_CLASS_VERSION, params 0x1373
[2017-04-24T15:18:40.234Z] Command[457] start: sendData
[2017-04-24T15:18:40.239Z] ProcessSendData[434]: To node: 53 with data: 0x861373 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:18:40.274Z] Command[457] end: sendData
[2017-04-24T15:18:40.297Z] Node[53]: Received application command for COMMAND_CLASS_VERSION, data: 0x147301
[2017-04-24T15:18:40.299Z] Command[449] status: addNode, status: INTERVIEW_CC_8_OF_13
[2017-04-24T15:18:40.300Z] Node[53]: sendData to COMMAND_CLASS_VERSION, params 0x1359
[2017-04-24T15:18:40.301Z] Command[458] start: sendData
[2017-04-24T15:18:40.309Z] ProcessSendData[435]: To node: 53 with data: 0x861359 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:18:40.346Z] Command[458] end: sendData
[2017-04-24T15:18:40.365Z] Node[53]: Received application command for COMMAND_CLASS_VERSION, data: 0x145901
[2017-04-24T15:18:40.371Z] Command[449] status: addNode, status: INTERVIEW_CC_9_OF_13
[2017-04-24T15:18:40.372Z] Node[53]: sendData to COMMAND_CLASS_VERSION, params 0x135a
[2017-04-24T15:18:40.373Z] Command[459] start: sendData
[2017-04-24T15:18:40.377Z] ProcessSendData[436]: To node: 53 with data: 0x86135a and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:18:40.409Z] Command[459] end: sendData
[2017-04-24T15:18:40.436Z] Node[53]: Received application command for COMMAND_CLASS_VERSION, data: 0x145a01
[2017-04-24T15:18:40.438Z] Command[449] status: addNode, status: INTERVIEW_CC_10_OF_13
[2017-04-24T15:18:40.439Z] Node[53]: sendData to COMMAND_CLASS_VERSION, params 0x135b
[2017-04-24T15:18:40.440Z] Command[460] start: sendData
[2017-04-24T15:18:40.447Z] ProcessSendData[437]: To node: 53 with data: 0x86135b and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:18:40.480Z] Command[460] end: sendData
[2017-04-24T15:18:40.502Z] Node[53]: Received application command for COMMAND_CLASS_VERSION, data: 0x145b02
[2017-04-24T15:18:40.505Z] Command[449] status: addNode, status: INTERVIEW_CC_11_OF_13
[2017-04-24T15:18:40.505Z] Node[53]: sendData to COMMAND_CLASS_VERSION, params 0x135b
[2017-04-24T15:18:40.506Z] Command[461] start: sendData
[2017-04-24T15:18:40.511Z] ProcessSendData[438]: To node: 53 with data: 0x86135b and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:18:40.549Z] Command[461] end: sendData
[2017-04-24T15:18:40.569Z] Node[53]: Received application command for COMMAND_CLASS_VERSION, data: 0x145b02
[2017-04-24T15:18:40.572Z] Command[449] status: addNode, status: INTERVIEW_CC_12_OF_13
[2017-04-24T15:18:40.573Z] Node[53]: sendData to COMMAND_CLASS_VERSION, params 0x1384
[2017-04-24T15:18:40.573Z] Command[462] start: sendData
[2017-04-24T15:18:40.579Z] ProcessSendData[439]: To node: 53 with data: 0x861384 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:18:40.722Z] Command[462] end: sendData
[2017-04-24T15:18:40.742Z] Node[53]: Received application command for COMMAND_CLASS_VERSION, data: 0x148402
[2017-04-24T15:18:40.744Z] Command[449] status: addNode, status: INTERVIEW_CC_13_OF_13
[2017-04-24T15:18:41.135Z] Command[449] status: addNode, status: INTERVIEW_MANUFACTURER_SPECIFIC
[2017-04-24T15:18:41.137Z] Node[53]: sendData to COMMAND_CLASS_MANUFACTURER_SPECIFIC, params 0x0404
[2017-04-24T15:18:41.137Z] Command[463] start: sendData
[2017-04-24T15:18:41.147Z] ProcessSendData[440]: To node: 53 with data: 0x720404 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:18:41.202Z] Command[463] end: sendData
[2017-04-24T15:18:44.224Z] Node[53]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:18:44.229Z] Node[53]: Marked as online
[2017-04-24T15:18:45.260Z] Node[53]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:18:46.234Z] Node[53]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:18:47.234Z] Node[53]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:18:48.234Z] Node[53]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:18:48.956Z] Node[19]: Received application command for COMMAND_CLASS_CRC_16_ENCAP, data: 0x013105030a004d9ef8
[2017-04-24T15:18:48.972Z] Node[19]: Decapsulated frame from COMMAND_CLASS_CRC_16_ENCAP to COMMAND_CLASS_SENSOR_MULTILEVEL, data 0x05030a004d
[2017-04-24T15:18:49.237Z] Node[53]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:18:50.243Z] Node[53]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:18:51.247Z] Node[53]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:18:52.246Z] Node[53]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:18:54.568Z] Node[53]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:18:56.058Z] Node[53]: Received application command for COMMAND_CLASS_BATTERY, data: 0x03ff
[2017-04-24T15:19:11.137Z] Command[449] end: addNode
```

**Comment by SebastianWeb** [comment]
Same here:

> [2017-04-24T15:23:41.375Z] Command[1611] start: addNode
[2017-04-24T15:23:41.429Z] Command[1611] status: addNode, status: ADD_NODE_STATUS_LEARN_READY
[2017-04-24T15:23:43.838Z] Command[1611] status: addNode, status: ADD_NODE_STATUS_NODE_FOUND
[2017-04-24T15:23:44.482Z] Command[1611] status: addNode, status: ADD_NODE_STATUS_ADDING_CONTROLLER
[2017-04-24T15:23:46.276Z] Command[1611] status: addNode, status: ADD_NODE_STATUS_PROTOCOL_DONE
[2017-04-24T15:23:46.374Z] Command[1611] status: addNode, status: ADD_NODE_STATUS_DONE
[2017-04-24T15:23:46.822Z] Command[1611] status: addNode, status: INTERVIEW_CC
[2017-04-24T15:23:46.823Z] Node[58]: sendData to COMMAND_CLASS_VERSION, params 0x135e
[2017-04-24T15:23:46.824Z] Command[1612] start: sendData
[2017-04-24T15:23:46.829Z] ProcessSendData[1607]: To node: 58 with data: 0x86135e and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:23:46.871Z] Command[1612] end: sendData
[2017-04-24T15:23:46.891Z] Node[58]: Received application command for COMMAND_CLASS_VERSION, data: 0x145e02
[2017-04-24T15:23:46.900Z] Command[1611] status: addNode, status: INTERVIEW_CC_1_OF_13
[2017-04-24T15:23:46.901Z] Node[58]: sendData to COMMAND_CLASS_VERSION, params 0x1385
[2017-04-24T15:23:46.901Z] Command[1613] start: sendData
[2017-04-24T15:23:46.908Z] ProcessSendData[1608]: To node: 58 with data: 0x861385 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:23:46.948Z] Command[1613] end: sendData
[2017-04-24T15:23:46.979Z] Node[58]: Received application command for COMMAND_CLASS_VERSION, data: 0x148502
[2017-04-24T15:23:46.982Z] Command[1611] status: addNode, status: INTERVIEW_CC_2_OF_13
[2017-04-24T15:23:46.983Z] Node[58]: sendData to COMMAND_CLASS_VERSION, params 0x1372
[2017-04-24T15:23:46.983Z] Command[1614] start: sendData
[2017-04-24T15:23:46.989Z] ProcessSendData[1609]: To node: 58 with data: 0x861372 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:23:47.017Z] Command[1614] end: sendData
[2017-04-24T15:23:47.045Z] Node[58]: Received application command for COMMAND_CLASS_VERSION, data: 0x147202
[2017-04-24T15:23:47.047Z] Command[1611] status: addNode, status: INTERVIEW_CC_3_OF_13
[2017-04-24T15:23:47.048Z] Node[58]: sendData to COMMAND_CLASS_VERSION, params 0x1321
[2017-04-24T15:23:47.048Z] Command[1615] start: sendData
[2017-04-24T15:23:47.054Z] ProcessSendData[1610]: To node: 58 with data: 0x861321 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:23:47.085Z] Command[1615] end: sendData
[2017-04-24T15:23:47.112Z] Node[58]: Received application command for COMMAND_CLASS_VERSION, data: 0x142101
[2017-04-24T15:23:47.115Z] Command[1611] status: addNode, status: INTERVIEW_CC_4_OF_13
[2017-04-24T15:23:47.115Z] Node[58]: sendData to COMMAND_CLASS_VERSION, params 0x1384
[2017-04-24T15:23:47.116Z] Command[1616] start: sendData
[2017-04-24T15:23:47.121Z] ProcessSendData[1611]: To node: 58 with data: 0x861384 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:23:47.148Z] Command[1616] end: sendData
[2017-04-24T15:23:47.176Z] Node[58]: Received application command for COMMAND_CLASS_VERSION, data: 0x148402
[2017-04-24T15:23:47.179Z] Command[1611] status: addNode, status: INTERVIEW_CC_5_OF_13
[2017-04-24T15:23:47.179Z] Node[58]: sendData to COMMAND_CLASS_VERSION, params 0x1386
[2017-04-24T15:23:47.180Z] Command[1617] start: sendData
[2017-04-24T15:23:47.187Z] ProcessSendData[1612]: To node: 58 with data: 0x861386 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:23:47.216Z] Command[1617] end: sendData
[2017-04-24T15:23:47.240Z] Node[58]: Received application command for COMMAND_CLASS_VERSION, data: 0x148602
[2017-04-24T15:23:47.243Z] Command[1611] status: addNode, status: INTERVIEW_CC_6_OF_13
[2017-04-24T15:23:47.243Z] Node[58]: sendData to COMMAND_CLASS_VERSION, params 0x1380
[2017-04-24T15:23:47.244Z] Command[1618] start: sendData
[2017-04-24T15:23:47.249Z] ProcessSendData[1613]: To node: 58 with data: 0x861380 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:23:47.276Z] Command[1618] end: sendData
[2017-04-24T15:23:47.309Z] Node[58]: Received application command for COMMAND_CLASS_VERSION, data: 0x148001
[2017-04-24T15:23:47.311Z] Command[1611] status: addNode, status: INTERVIEW_CC_7_OF_13
[2017-04-24T15:23:47.312Z] Node[58]: sendData to COMMAND_CLASS_VERSION, params 0x1373
[2017-04-24T15:23:47.313Z] Command[1619] start: sendData
[2017-04-24T15:23:47.318Z] ProcessSendData[1614]: To node: 58 with data: 0x861373 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:23:47.346Z] Command[1619] end: sendData
[2017-04-24T15:23:47.371Z] Node[58]: Received application command for COMMAND_CLASS_VERSION, data: 0x147301
[2017-04-24T15:23:47.373Z] Command[1611] status: addNode, status: INTERVIEW_CC_8_OF_13
[2017-04-24T15:23:47.374Z] Node[58]: sendData to COMMAND_CLASS_VERSION, params 0x1359
[2017-04-24T15:23:47.374Z] Command[1620] start: sendData
[2017-04-24T15:23:47.380Z] ProcessSendData[1615]: To node: 58 with data: 0x861359 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:23:47.408Z] Command[1620] end: sendData
[2017-04-24T15:23:47.432Z] Node[58]: Received application command for COMMAND_CLASS_VERSION, data: 0x145901
[2017-04-24T15:23:47.435Z] Command[1611] status: addNode, status: INTERVIEW_CC_9_OF_13
[2017-04-24T15:23:47.436Z] Node[58]: sendData to COMMAND_CLASS_VERSION, params 0x135a
[2017-04-24T15:23:47.436Z] Command[1621] start: sendData
[2017-04-24T15:23:47.442Z] ProcessSendData[1616]: To node: 58 with data: 0x86135a and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:23:47.470Z] Command[1621] end: sendData
[2017-04-24T15:23:47.496Z] Node[58]: Received application command for COMMAND_CLASS_VERSION, data: 0x145a01
[2017-04-24T15:23:47.499Z] Command[1611] status: addNode, status: INTERVIEW_CC_10_OF_13
[2017-04-24T15:23:47.500Z] Node[58]: sendData to COMMAND_CLASS_VERSION, params 0x135b
[2017-04-24T15:23:47.500Z] Command[1622] start: sendData
[2017-04-24T15:23:47.506Z] ProcessSendData[1617]: To node: 58 with data: 0x86135b and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:23:47.533Z] Command[1622] end: sendData
[2017-04-24T15:23:47.561Z] Node[58]: Received application command for COMMAND_CLASS_VERSION, data: 0x145b02
[2017-04-24T15:23:47.563Z] Command[1611] status: addNode, status: INTERVIEW_CC_11_OF_13
[2017-04-24T15:23:47.564Z] Node[58]: sendData to COMMAND_CLASS_VERSION, params 0x135b
[2017-04-24T15:23:47.565Z] Command[1623] start: sendData
[2017-04-24T15:23:47.570Z] ProcessSendData[1618]: To node: 58 with data: 0x86135b and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:23:47.597Z] Command[1623] end: sendData
[2017-04-24T15:23:47.625Z] Node[58]: Received application command for COMMAND_CLASS_VERSION, data: 0x145b02
[2017-04-24T15:23:47.628Z] Command[1611] status: addNode, status: INTERVIEW_CC_12_OF_13
[2017-04-24T15:23:47.629Z] Node[58]: sendData to COMMAND_CLASS_VERSION, params 0x1384
[2017-04-24T15:23:47.629Z] Command[1624] start: sendData
[2017-04-24T15:23:47.634Z] ProcessSendData[1619]: To node: 58 with data: 0x861384 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:23:47.661Z] Command[1624] end: sendData
[2017-04-24T15:23:47.690Z] Node[58]: Received application command for COMMAND_CLASS_VERSION, data: 0x148402
[2017-04-24T15:23:47.692Z] Command[1611] status: addNode, status: INTERVIEW_CC_13_OF_13
[2017-04-24T15:23:48.060Z] Command[1611] status: addNode, status: INTERVIEW_MANUFACTURER_SPECIFIC
[2017-04-24T15:23:48.062Z] Node[58]: sendData to COMMAND_CLASS_MANUFACTURER_SPECIFIC, params 0x0404
[2017-04-24T15:23:48.062Z] Command[1625] start: sendData
[2017-04-24T15:23:48.067Z] ProcessSendData[1620]: To node: 58 with data: 0x720404 and txOptions: ACK,AUTO_ROUTE,EXPLORE
[2017-04-24T15:23:48.165Z] Command[1625] end: sendData
[2017-04-24T15:23:51.167Z] Node[58]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:23:51.179Z] Node[58]: Marked as online
[2017-04-24T15:23:52.141Z] Node[58]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:23:53.145Z] Node[58]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:23:54.145Z] Node[58]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:23:55.145Z] Node[58]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:23:56.149Z] Node[58]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:23:57.149Z] Node[58]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:23:58.151Z] Node[58]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:23:59.141Z] Node[58]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:24:00.152Z] Node[58]: Received application command for COMMAND_CLASS_WAKE_UP, data: 0x07
[2017-04-24T15:24:18.062Z] Command[1611] end: addNode
[2017-04-24T15:24:20.234Z] Command[1629] start: addNodeAbort
[2017-04-24T15:24:20.240Z] Command[1629] end: addNodeAbort

**Comment by TedTolboom** [comment]
@jeroenvollenbrock despite that the interview failed, Homey indicates that no Node has been found and the Z-wave map shows a node added as unknown device... 

This node does report battery reports on every press of a button like specified in the manual..
[2017-04-24T19:37:47.350Z] Node[57]: Received application command for COMMAND_CLASS_BATTERY, data: 0x03ff

In addition, I do wonder why there are in the INTERVIEW_MANUFACTURER_SPECIFIC section consistently 9 (or 10) Wake_up reports at exactly 1 second gap... after which the addNode is aborted... 

Based on what condition is the interview aborted; no communication of after a specified duration?

**Comment by TedTolboom** [comment]
@jeroenvollenbrock some additional notes:

Even when the interview of the Scene master gets stuck at INTERVIEW_MANUFACTURER_SPECIFIC, the node is added to the network, only the device (incl. its driver and apps) not..

I tried to extend the wake-up period from the Scene Master, but it does not resolve the issue.

Based on the RAW data input, I'm able to get the Manufacturer specific data from the scene master, after decoding it it provides the ID's also added to the app...
```
[2017-04-26T21:51:05.110Z] Node[57]: Received application command for COMMAND_CLASS_MANUFACTURER_SPECIFIC, 
data: 0x05525400018510
0x05		= MANUFACTURER_SPECIFIC_REPORT
5254 = 21076	= Manufacturer ID
0001 = 1	= Product Type ID
8510 = 34064	= Product ID
```
Is it possible that something went wrong with processing this command? 
I did see more issues on the forum where the interview get's stuck on this step.

Working with the RAW data further, I'm able to SET the association to the first group of this device; so this also seems to work.
```
[2017-04-26T22:48:04.411Z] Node[57]: Received application command for COMMAND_CLASS_ASSOCIATION, 
data: 0x0301010001
03 		= ASSOCIATION_REPORT
01 	= 1	= Grouping Identifier
01 	= 1	= Max Nodes Supported
00	= 0	= Reports to Follow
01	=  1	= NodeID 1
```
After setting the association, I am able to receive the CENTRAL_SCENE reports from this device... 
But still no driver / app loaded...
